packages/serverpod/lib/src/database/database_result.dart

```diff
@@ -7,9 +7,13 @@ Map<String, dynamic>? resolvePrefixedQueryRow(
   Map<String, Map<String, dynamic>> rawRow,
   Map<String, Map<int, List<dynamic>>> resolvedListRelations, {
   Include? include,
+  bool viewTable = false,
 }) {
+  assert(!viewTable || (viewTable && rawRow.containsKey('')),
+      'Invalid rawRow for viewTable=true');
```

Comment:
Suggestion: Chage to a throw.

Asserts only work in debug environments so this method would be run if the rawRow had the wrong format in a production environment.

Also, I think this error message needs to be inproved to help any future developer understand what has gone wrong.